ecosystem/typescript/sdk/src/token_types.ts

```diff
@@ -0,0 +1,42 @@
+export interface TokenData {
```

Comment:
These are not generated any more?

Reply:
Yeah, because these types were never actually part of the API, they were added to the spec by hand. These fall more under the types that you're working on.